app/assets/stylesheets/application.css.scss

```diff
@@ -74,129 +69,41 @@ ul {
   width: 300px;
 }
 
-/* drag and drop character pics */
-.layout {
-    height: 100%;
-
-    * {
-        box-sizing: border-box;
-    }
-
-    body {
-        width: 100%;
-        height: 100%;
-        padding: 20px;
-        color: #111;
-        background-image: radial-gradient(ellipse farthest-corner at center, #7abcff 0%,#4096ee 100%);
-    }
-
-    h1 {
-        font-family: Lobster, Arial;
-        text-align: center;
-    }
-
-    .btn {
-        position: absolute;
-        right: 5px;
-        bottom: 5px;
-        z-index: 999;
-        padding: 6px 10px;
-        font-family: "Ubuntu";
-        font-size: 16px;
-        color: #fff;
-        background-image: linear-gradient(to bottom, #c92437 0%, darken(#c92437, 10%) 100%);
-        border: 0;
-        border-radius: 5px;
-        box-shadow: 1px 1px 2px rgba(0,0,0,.3);
-    }
-}
-
-// Image Crop CSS
-
-.ic-container {
-    position: relative;
-}
+$dropdown-inline-display: inline-block !important;
+$dropdown-inline-width: 30% !important;
 
-.ic-overlay-n,
-.ic-overlay-e,
-.ic-overlay-s,
-.ic-overlay-w {
-    position: absolute;
-    background-color: #000;
-    opacity: 0.7;
-}
-
-.ic-overlay-n {
-    top: 0;
-}
-
-.ic-overlay-e {
-    top: 0;
-    right: 0;
-    bottom: 0;
-}
-
-.ic-overlay-s {
-    bottom: 0;
-}
-
-.ic-overlay-w {
-    top: 0;
-    bottom: 0;
-    left: 0;
-}
-
-.ic-resize-handle-ne,
-.ic-resize-handle-se,
-.ic-resize-handle-nw,
-.ic-resize-handle-sw {
-    position: absolute;
-    width: 10px;
-    height: 10px;
-    background: #c92437;
-    z-index: 999;
-}
-
-.ic-resize-handle-nw {
-    top: -5px;
-    left: -5px;
-    cursor: nw-resize;
-}
-
-.ic-resize-handle-sw {
-    bottom: -5px;
-    left: -5px;
-    cursor: sw-resize;
+.home-header {
+  background-image: image-url("SpookyHouseHeader.gif");
+  background-position: no-repeat center center fixed;
+  background-size: cover;
+  height: 500px;
 }
 
-.ic-resize-handle-ne {
-    top: -5px;
-    right: -5px;
-    cursor: ne-resize;
+.about {
+  background:rgba(100,100,100,0.8);
+  color: black;
+  margin: 75px;
+  padding: 10px;
 }
 
-.ic-resize-handle-se {
-    bottom: -5px;
-    right: -5px;
-    cursor: se-resize;
+.joingame {
+  background-image: image-url("joinagame.gif");
+  background-size: cover;
+  height: 400px;
 }
 
-.ic-crop-marker {
-    position: absolute;
-    z-index: 999;
-    border: solid 2px rgba(201,36,55,.5);
-    cursor: move;
+.gameblock {
+  padding: 8% 0px;
+  height: 150px;
+  width: 150px;
 }
 
-.ic-move-handle {
-    position: absolute;
-    top: 0;
-    right: 0;
-    bottom: 0;
-    left: 0;
+div.gameblock:nth-child(even){
+  background:rgba(0,0,0,0.5);
```

Comment:
Commas in function arguments should be followed by a single space<br>Colon after property should be followed by one space